features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/edit/RoomDetailsEditPresenter.kt

```diff
@@ -61,41 +64,56 @@ class RoomDetailsEditPresenter @Inject constructor(
         val cameraPermissionState = cameraPermissionPresenter.present()
         val roomSyncUpdateFlow = room.syncUpdateFlow.collectAsState()
 
-        // Since there is no way to obtain the new avatar uri after uploading a new avatar,
-        // just erase the local value when the room field has changed
-        var roomAvatarUri by rememberSaveable(room.avatarUrl) { mutableStateOf(room.avatarUrl?.toUri()) }
+        val roomAvatarUri = room.avatarUrl()?.toUri()
+        var roomAvatarUriEdited by rememberSaveable { mutableStateOf<Uri?>(null) }
+        LaunchedEffect(roomAvatarUri) {
+            // Every time the roomAvatar change (from sync), we can set the new avatar.
```

Comment:
Do we really want to have live values here? I'd expect to be static

Reply:
Yes, I think it's fine, having the value change during edition should not happen a lot, just during our cheeky tests :)